pkgs/development/libraries/decaf/default.nix

```diff
@@ -0,0 +1,38 @@
+{ cmake
+, fetchFromGitLab
+, lib
+, python3
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "decaf";
+  version = "linphone-4.4.1";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    python3
+  ];
+
+  src = fetchFromGitLab {
+    domain = "gitlab.linphone.org";
```

Comment:
If you're pulling decaf from your GitLab this should IMO be called bc-decaf or something. You're not packaging upstream decaf.